tests/testthat/test-unit-predict.R

```diff
@@ -0,0 +1,47 @@
+prep_obs <- as_latent_individual(sim_obs)
+fit <- epidist(prep_obs)
```

Comment:
I don't want to have to run this every time, but I want to always have `prep` available for these tests. What to do? Can't have it here because it'll error on CRAN

Reply:
In epinowcast we store an example in inst and use that which is slightly annoying but works.

Could have it in setup.R and have these tests only run if not on CRAN

Look at what brms does?

Reply:
It has `brms:::brmsfit_example1` (so putting examples as unexported data). It also constructs the `prep` object including using functions like `brms:::add_dummy_draws`

Reply:
Maybe I'm wrong that this means putting examples in unexported data

Reply:
I've done the `inst/extdata` approach for now but I think this needs further thought. We have an issue for "reduce number of times model is fit in tests"